EarlyBinding/src/EXPRESS/EXPRESSModel.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+    This file is part of Expresso, a simple early binding generator for EXPRESS.
+	Copyright (c) 2016 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+    BlueFramework is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License Version 3
+    as published by the Free Software Foundation.
+
+    BlueFramework is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
```

Comment:
We maybe should fix the license header to be accurate.

Reply:
Agreed.